nixos/modules/services/networking/searx.nix

```diff
@@ -158,6 +191,17 @@ in
   ###### implementation
 
   config = mkIf cfg.enable {
+    assertions = [
+      {
+        assertion = cfg.redisCreateLocally -> (cfg.settings.redis.url == "redis://localhost:6379");
+        message = "When services.searx.redisCreateLocally is enabled, you need to use redis://localhost:6379 as a cache server.";
+      }
+      {
+        assertion = (cfg.limiterSettings != { }) -> cfg.package.pname == "searxng";
+        message = "services.searx.limiterSettings requires services.searx.package to be searxng.";
+      }
```

Comment:
Maybe we can also switch the default at some point to searxng

Reply:
Yeah, searx development has kind of halted and broken engines are piling up.